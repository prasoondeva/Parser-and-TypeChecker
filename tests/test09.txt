{
	x , z : INT;
	y : STRING;
	x = 1;
	y = "b";
	z = + x y;
}