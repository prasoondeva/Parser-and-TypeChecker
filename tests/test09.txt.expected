TYPE MISMATCH 6 C3